content/work/quarqnet.md

```diff
@@ -3,7 +3,8 @@ card: large
 title: QuarqNet
 banner: QuarqNet Application
 sub: Real-time telemetry & sharing for athletes
-url: http://quarqnet.com
+links:
+  site: https://www.quarqnet.com/
```

Comment:
@mirisuzanne Was there a reason not to link off to these sites (also Emmy's, MetaDeploy, Quarqrace)?

Reply:
nope, may as well link anything that's live & public in some form